decidim-core/lib/decidim/hashtagging.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Decidim
+  class Hashtagging < ApplicationRecord
+    self.table_name = "decidim_hashtaggings"
+
+    belongs_to :decidim_hashtag, class_name: "Decidim::Hashtag"
+    belongs_to :decidim_hashtaggable, class_name: "Decidim::Hashtaggable", polymorphic: true
```

Comment:
We should check that the hastaggable and the hashtag belong to the same organization.